tests/lib/cli.js

```diff
@@ -287,6 +287,17 @@ describe("cli", () => {
         });
     });
 
+    describe("when given an async formatter path", () => {
+        it("should execute without any errors", async () => {
+            const formatterPath = getFixturePath("async-formatter.js");
```

Comment:
```suggestion
            const formatterPath = getFixturePath("formatters", "async-formatter.js");
```

Can we move this file to `fixtures/formatters/`, where the other formatters are?

Reply:
I did it before, but other tests failed, not sure why

Reply:
done @mdjermanovic

Reply:
It appears that we're using files in `fixtures/formatters/` as source code files in some tests, so whenever we add a fixture formatter to test how the formatters feature works, we have to update some unrelated tests that are linting those files.

I'll fix this in a follow-up.